src/data/faq.json

```diff
@@ -3362,6 +3362,11 @@
                     "anchor": "science_blog",
                     "description": "The Corona-Warn-App Science Blog covers various scientific topics related to the Corona-Warn-App, such as its effectiveness, benefits, as well as results of scientific evaluations conducted by the Robert Koch Institute. See <a href='../../science'>Science-Blog</a>."
                 },
+                {
+                    "term": "Self-test",
+                    "anchor": "selftest",
+                    "description": "Self-tests are available free of charge and can be used safely even by untrained individuals. For the evaluation of a self-test, which will be done within 15 to 30 minutes, there is no need for a laboratory. Therefore, you do not have to go to a doctor, the health department or a testing station, but can test yourself comfortably at home. However, a self-test is only reliable if it has been carried out correctly. Because this is difficult to detect, self-test results are not sufficient for testing yourself free from a prescribed quarantine, nor for confirmation of infection. In both cases, a certified <a href='#glossary_rapid_antigen_test'>rapid antigen test</a> is necessary."

```

Comment:
I'm not sure if "free of charge" is the correct term to use here, as you have to buy these tests and they are not for free.

Reply:
The translation for "freiverlkäuflich" would be "freely available", not "free of charge"

Reply:
Thanks for the information. It has been changed to "freely available" in https://github.com/corona-warn-app/cwa-website/pull/3001/commits/6fae75ec81cc132a4e6e6f29927df3283fa16916